aws-android-sdk-s3/src/main/java/com/amazonaws/mobileconnectors/s3/transferutility/TransferConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.amazonaws.mobileconnectors.s3.transferutility;
+
+import com.amazonaws.mobileconnectors.s3.receiver.NetworkInfoReceiver;
+
+/**
+ * Created by <a href="mailto:Dmitry.Gorohov@ebuilder.com">Dmitry Gorohov</a>
+ */
+public class TransferConfiguration {
```

Comment:
We have introduced a class called TransferUtilityOptions with two parameters. Can you add NetworkConnectionType as a third parameter to the options?